.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -1,71 +1,55 @@
-const {exec} = require('child_process');
-const fs = require('fs');
 const core = require('@actions/core');
 const github = require('@actions/github');
-
-// Use Github Actions' default environment variables to get repo information
-// https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
-const [repoOwner, repoName] = process.env.GITHUB_REPOSITORY.split('/');
+const functions = require('./functions');
 
 const MAX_RETRIES = 10;
```

Comment:
I know you wouldn't have known this when you were creating the PR, but you shouldn't need any retry logic anymore (should be more obvious once you merge master into your branch).

Reply:
I checked you changes, no problem.